### PR TITLE
README - reposition this repo

### PR DESCRIPTION
**Pull request by hamishwillee**
@julianoes @JonasVautherin We need to think about the README again. 
1. We don't need the class overview - should point to CPP lib docs for this (reduces maintenance).
2. This is just the core library, but if you discover this repo then you'd think that this is the only API.

So what this does is provide an entry point to the MAVSDK ecosystem assuming you find this repo first. It describes what MAVSDK is, points to our docs and other repos for other things.

Very happy for you to improve this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,51 +6,44 @@
...
+- [MAVSDK-Python](https://github.com/mavlink/MAVSDK-Python) - MAVSDK client for Python
+- [MAVSDK-Swift](https://github.com/mavlink/MAVSDK-Swift) - MAVSDK client for Swift.
+- [MAVSDK-Java](https://github.com/mavlink/MAVSDK-Java) - MAVSDK client for Java.
+- [MAVSDK-JavaScript](https://github.com/mavlink/MAVSDK-JavaScript) - JS Wrapper for MAVSDK using grpc-web to generate static http client compatible with Envoy Proxy
+- [Other repos](https://github.com/mavlink?q=MAVSDK) - Docs, examples, etc.
 
-## API Overview
 
-API consumers use `MAVSDK` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
+## Status
```
Comment (hamishwillee) [inline README.md]: @JonasVautherin Is the status of each of these correct? What dates should we put in for each of the libraries. I assumed "first usable version".

File: README.md [hunk 2]
```diff
@@ -6,51 +6,44 @@
...
 
-The links below take you to the respective header files:
+- C++ core library (2016). Reliable and stable - used in production.
+- MAVSDK-Swift (): ?: Used in production.
+- MAVSDK-Python (2019): First PyPi release 2019.
+- MAVSDK-Java (): Prototype
+- MAVSDK-JavaScript (2019). Prototype
 
-- [mavsdk](src/core/mavsdk.h): set up connection, discover devices
```
Comment (hamishwillee) [inline README.md]: Removed all this. They can get from the guide.

File: README.md [hunk 3]
```diff
@@ -6,51 +6,44 @@
...
+- [Other repos](https://github.com/mavlink?q=MAVSDK) - Docs, examples, etc.
 
-## API Overview
 
-API consumers use `MAVSDK` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
+## Status
 
-The links below take you to the respective header files:
+- C++ core library (2016). Reliable and stable - used in production.
+- MAVSDK-Swift (): ?: Used in production.
```
Comment (JonasVautherin) [inline README.md]:
```suggestion
- MAVSDK-Swift (2018): Reliable and stable - used in production.
```

File: README.md [hunk 4]
```diff
@@ -6,51 +6,44 @@
...
+- MAVSDK-Python (2019): First PyPi release 2019.
+- MAVSDK-Java (): Prototype
+- MAVSDK-JavaScript (2019). Prototype
```
Comment (JonasVautherin) [inline README.md]:
```suggestion
- MAVSDK-JavaScript (2019). Proof of concept.
```

File: README.md [hunk 5]
```diff
@@ -6,51 +6,44 @@
...
-## API Overview
 
-API consumers use `MAVSDK` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
+## Status
 
-The links below take you to the respective header files:
+- C++ core library (2016). Reliable and stable - used in production.
+- MAVSDK-Swift (): ?: Used in production.
+- MAVSDK-Python (2019): First PyPi release 2019.
+- MAVSDK-Java (): Prototype
```
Comment (JonasVautherin) [inline README.md]:
```suggestion
- MAVSDK-Java (2019): Prototype.
```

File: README.md (hunk 1 above)
Comment (JonasVautherin) [reply README.md]: I made some suggestions. Quick note: when we say "reliable and stable", we mean that it doesn't crash and can be used in production, right? As mentioned last week, the API is not completely stable yet, so (fairly small) changes may come and break the build.

Comment (hamishwillee) [reply README.md]: Yes, that is what I mean by "stable" in this case.

**Comment by hamishwillee** [comment]
Thanks for review @JonasVautherin - accepted all your changes thanks! Merging now.

**Pull request merged**